setup.py

```diff
@@ -1,10 +1,17 @@
-from distutils.core import setup
-from setuptools import setup
+from setuptools import setup, Extension
+from Cython.Distutils import build_ext
+from Cython.Build import cythonize
 
-VERSION = "0.1.0"
+
+ext_modules = cythonize(Extension("ndmg.graph.zindex", # the extension name
+                                  sources=["ndmg/graph/zindex.pyx"],
+                                  include_dirs=['.'],
+                                  language="c"))
+VERSION="0.2.0"
```

Comment:
This is inconsistent with the version in the `__init__.py` above. Make sure you consolidate them somehow so this problem doesn't keep popping up.